src/main/java/org/asciidoc/intellij/ui/FormatterFactory.java

```diff
@@ -0,0 +1,21 @@
+package org.asciidoc.intellij.ui;
+
+import javax.swing.text.NumberFormatter;
+import java.text.NumberFormat;
+
+import static java.lang.Integer.MAX_VALUE;
+
+public final class FormatterFactory {
+
+  public static NumberFormatter createIntegerFormatter() {
+    NumberFormatter formatter = new NumberFormatter(NumberFormat.getInstance());
+
+    formatter.setValueClass(Integer.class);
+    formatter.setMinimum(0);
```

Comment:
An image of size 0 will now show and users might be surprised to not see the result. What about 5 as a minimum size?